homeassistant/components/media_player/plex.py

```diff
@@ -62,10 +101,13 @@ def config_from_file(filename, config=None):
 
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """Setup the Plex platform."""
-    config = config_from_file(hass.config.path(PLEX_CONFIG_FILE))
-    if len(config):
+    # get config from plex.conf
+    file_config = config_from_file(hass.config.path(PLEX_CONFIG_FILE))
+    config.update(file_config)  # merge configs
```

Comment:
each plexdevice object created needs to access some of the optional config variables I added.  So I minimally need to pass the config to the object.  I see a line where the config object is being cleared and reset to config grabbed from plex.conf.  So I merged the 2 objects instead of having it be overwritten.  Let me know if that's necessary.